.eslintrc

```diff
@@ -20,6 +20,16 @@
     "import/ignore": [".ejs$", ".yml$", ".jsx", "node_modules"]
   },
   "rules": {
-    "lines-between-class-members": "off"
+    "lines-between-class-members": "off",
+    "global-require": "off"
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
```

Comment:
Question: I don't know that setting the sourceType here like this is appropriate since our library isn't setup as a module, though I know why you did it. Is there another option to fixing the `import` lint errors instead of saying this repo is a module which it's not?

Reply:
Actually, we don't need this anymore.